.github/workflows/ci.yml

```diff
@@ -1,37 +1,23 @@
 name: Continuous Integration
 
 on:
-  push:
   pull_request:
 
 jobs:
-  htmlproofer:
-    name: HTMLProofer
+  link-check:
+    name: Link Check
     runs-on: ubuntu-latest
     
     steps:
       - name: Checkout EIP Repository
         uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
-      
-      - name: Install OpenSSL
-        run: sudo apt-get update && sudo apt-get install -y libcurl4-openssl-dev
-    
-      - name: Install Ruby
-        uses: ruby/setup-ruby@08245253a76fa4d1e459b7809579c62bd9eb718a
-        with:
-          ruby-version: '2.6.0'
-          bundler-cache: true
         
-      - name: Build Website
-        run: |
-          bundle exec jekyll doctor
-          bundle exec jekyll build
-      
-      - name: HTML Proofer
-        run: bundle exec htmlproofer ./_site --check-html --check-opengraph --report-missing-names --log-level=:debug --assume-extension --empty-alt-ignore --timeframe=6w --disable-external
-      
-      - name: DNS Validator
-        run: bundle exec github-pages health-check
```

Comment:
What did this health-check do previously, and does the new tool do it or do you believe this is no longer necessary?

Reply:
It checked that the CNAME `eips.ethereum.org` actually pointed to `ethereum.github.io`. I assume the Ethereum foundation isn't going to rug us (and if they did, the last thing we would want is our CI failing because of it).

Reply:
I would worry less about being rugged, and more about someone accidentally breaking something when updating DNS records.

Do we gain some tangible benefit from removing this that makes it worth losing out on robustness?

Reply:
If anything, the above should run as a periodic check on the repo, instead of on PR.

Reply:
I agree that a periodic check would be better.  At the moment I don't believe we have the infrastructure to do such things though, and I would want to see a replacement put into place that gives us confidence that should something break the appropriate parties will be notified before removing this.  Maybe @jpitts knows if someone already has this sort of monitoring in place though and if so then we can remove this?

Reply:
> At the moment I don't believe we have the infrastructure to do such things though,

We do. I can write it if you want!